Makefile

```diff
@@ -164,6 +166,15 @@ generate: controller-gen
 	go generate ./...
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
+# Check and add (if missing) license header
+LICENSE_FILE=$(shell pwd)/LICENSE
+.PHONY: add-license 
+add-license: addlicense
+	@echo "Checking/Adding license..."
+	@$(ADDLICENSE) -v -f $(LICENSE_FILE) api internal 2>&1
```

Comment:
This line can be removed now, right? The one below it is doing more or less the same thing.

Reply:
Oh that was a mistake :D That should be replaced with the new one...